### PR TITLE
Add index to activationStatus and MultipleValidatorStatus response

### PR DESCRIPTION
**Pull request by 0xKiwi**
**What type of PR is this?**
API Improvement 

**What does this PR do? Why is it needed?**
This PR modifies the `activationStatus` and `multipleValidatorStatus` to also return the indice, for the validator to log out the indices where there is room for them.

`status.go` was also a bit messy, so I've reordered some parameters and removed some unneeded parameters.

### CODE REVIEW

File: beacon-chain/rpc/validator/status.go [hunk 1]
```diff
@@ -51,89 +54,99 @@ func (vs *Server) MultipleValidatorStatus(
...
+// validatorStatus searches for the requested validator's state and deposit to retrieve its inclusion estimate. Also returns the validators index.
+func (vs *Server) validatorStatus(
+	ctx context.Context,
+	headState *stateTrie.BeaconState,
+	pubKey []byte,
+) (*ethpb.ValidatorStatusResponse, uint64) {
 	ctx, span := trace.StartSpan(ctx, "validatorServer.validatorStatus")
 	defer span.End()
 
+	// Using farFutureEpoch as the default value for index, in case the validators index cannot be determined.
```
Comment (terencechain) [inline beacon-chain/rpc/validator/status.go:133]: It's confusing to use `farFutureEpoch` as default for index. One's unit is `Epoch` and one's unit is `ValidatorIndex`

File: beacon-chain/rpc/validator/status.go [hunk 2]
```diff
@@ -30,19 +30,22 @@ var errPubkeyDoesNotExist = errors.New("pubkey does not exist")
...
 	ctx context.Context,
-	req *ethpb.ValidatorStatusRequest) (*ethpb.ValidatorStatusResponse, error) {
+	req *ethpb.ValidatorStatusRequest,
+) (*ethpb.ValidatorStatusResponse, error) {
 	headState, err := vs.HeadFetcher.HeadState(ctx)
 	if err != nil {
 		return nil, status.Error(codes.Internal, "Could not get head state")
 	}
-	return vs.validatorStatus(ctx, req.PublicKey, headState), nil
+	vStatus, _ := vs.validatorStatus(ctx, headState, req.PublicKey)
```
Comment (rauljordan) [inline beacon-chain/rpc/validator/status.go:39]: Unhandled error

File: beacon-chain/rpc/validator/status.go (hunk 1 above)
Comment (rauljordan) [reply beacon-chain/rpc/validator/status.go:133]: Agreed, is there a better way we could do this, maybe through a boolean ?

File: beacon-chain/rpc/validator/status.go (hunk 2 above)
Comment (0xKiwi) [reply beacon-chain/rpc/validator/status.go:39]: That is the return indice, the function doesn't return an error.

File: beacon-chain/rpc/validator/status.go (hunk 1 above)
Comment (0xKiwi) [reply beacon-chain/rpc/validator/status.go:133]: Hmm, That would require changing the protobufs as well no? Not sure how to go about this...

Comment (terencechain) [reply beacon-chain/rpc/validator/status.go:133]: Don't name it `farFutureEpoch`, name it like:
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/forkchoice/protoarray/types.go#L45

**Pull request merged**